levenstein.py

```diff
@@ -1,2 +1,24 @@
-def dist(s1: str, s2: str) -> int:
-    return 0
\ No newline at end of file
+"""PY algoritm Levenshtein Distance"""
+
+def dist(strng1: str, strng2: str) -> int:
+    """
+    Levenshtein Distance function
+    :param strng1: first string
+    :param strng2: second string
+    :return: Levenshtein Distance
+    """
+    strng1_len, strng2_len = len(strng1), len(strng2)
+    if strng1_len < strng2_len:
+        strng1, strng2 = strng2, strng1
+        strng1_len, strng2_len = strng2_len, strng1_len
+    result = 0
+    for i in range(strng2_len):
```

Comment:
It's not enough to calculate distance. You need compare any character with any other